pkg/controller/v1beta1/inferenceservice/reconcilers/cabundleconfigmap/cabundle_configmap_reconciler.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2021 The KServe Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cabundleconfigmap
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	"github.com/kserve/kserve/pkg/constants"
+	"github.com/kserve/kserve/pkg/webhook/admission/pod"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"knative.dev/pkg/kmp"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("CaBundleConfigMapReconciler")
+
+type CaBundleConfigMapReconciler struct {
+	client client.Client
+	scheme *runtime.Scheme
+}
+
+func NewCaBundleConfigMapReconciler(client client.Client, scheme *runtime.Scheme) *CaBundleConfigMapReconciler {
+	return &CaBundleConfigMapReconciler{
+		client: client,
+		scheme: scheme,
+	}
+}
+
+func (c *CaBundleConfigMapReconciler) Reconcile(isvc *kservev1beta1.InferenceService) error {
+	log.Info("Reconciling CaBundleConfigMap")
+
+	isvcConfigMap := &corev1.ConfigMap{}
+	err := c.client.Get(context.TODO(), types.NamespacedName{Name: constants.InferenceServiceConfigMapName, Namespace: constants.KServeNamespace}, isvcConfigMap)
+	if err != nil {
+		log.Error(err, "Failed to find config map", "name", constants.InferenceServiceConfigMapName)
+		return err
+	}
+
+	storageInitializerConfig := &pod.StorageInitializerConfig{}
+	if storageInitializerConfigValue, ok := isvcConfigMap.Data["storageInitializer"]; ok {
+		err := json.Unmarshal([]byte(storageInitializerConfigValue), &storageInitializerConfig)
+		if err != nil {
+			panic(fmt.Errorf("Unable to unmarshall storage initializer json string due to %w ", err))
```

Comment:
```suggestion
			panic(fmt.Errorf("unable to unmarshal storage initializer json string due to %w ", err))
```

For errros, there is a golang pattern to use lower case.

Reply:
updated